consul_kv_test.go

```diff
@@ -0,0 +1,39 @@
+package onlinelab
+
+import "testing"
+
+func TestSetConfig(t *testing.T) {
```

Comment:
Are these tests depend on a living Consul service? Can they be executed lonely without starting other service manually?
Make sure all your unit tests can be executed automatically.